aspects/schema.go

```diff
@@ -138,6 +142,8 @@ func (s *StorageSchema) parse(raw json.RawMessage) (parser, error) {
 		if err := schema.parseConstraints(schemaDef); err != nil {
 			return nil, err
 		}
+	} else if schema.mustConstrain() {
+		return nil, fmt.Errorf(`cannot parse %q: must be map definition with constraints`, typ)
```

Comment:
s/map definition/schema definition map/ perhaps,  otherwise it makes it sound it's about the map type?